torchvision/prototype/transforms/_type_conversion.py

```diff
@@ -20,19 +21,18 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
 
 
 class LabelToOneHot(Transform):
+    _transformed_types = (features.Label,)
```

Comment:
Good catch for using the new idiom. We should look for other transforms that use old styles and refactor.